doc/classes/ItemList.xml

```diff
@@ -239,6 +242,11 @@
 			<argument index="1" name="custom_bg_color" type="Color">
 			</argument>
 			<description>
+				Sets the background color of [code]idx[/code] item to the specified [Color].
+				[codeblock]
+					var someString = "Some text"
```

Comment:
As per the GDScript style guide, variables should use `snake_case`, so this should be `some_string`: https://docs.godotengine.org/en/3.1/getting_started/scripting/gdscript/gdscript_styleguide.html#functions-and-variables
There should not be an additional indentation level after `[codeblock]`. If you do add a function/if/etc. which requires indentation, you can then use 4 spaces for the GDScript indentation in the code block.

Reply:
The tabs were actually meant for the XML to be easier readable. I didn't expect them to be visible at the user.